sda-commons-server-mongo-testing/src/main/java/org/sdase/commons/server/mongo/testing/MongoDbRule.java

```diff
@@ -156,6 +157,7 @@ private static IDownloadConfig createDownloadConfig() {
         downloadConfigBuilder.downloadPath(embeddedMongoDownloadPath);
       }
 
+      downloadConfigBuilder.artifactStorePath(new UUIDDir());
```

Comment:
Just as a warning and because I know you are not working on Mac: If the artifact store is the location where the executable is stored, that we should change that per test. Because that would mean that the Mac users have to agree to the firewall settings on every class. That's the reason for having only a single location for it.

We already have a fix for that, but that one wasn't merged because the dead letter handling branch was closed. Maybe that helps you: https://github.com/SDA-SE/sda-dropwizard-commons/commit/4c0757ce91a0d3062d08452af6db1361a84fec60